index.php

```diff
@@ -23,33 +23,35 @@
 $templates = new Templates();
 $page = $templates->getPageUrl();
 $data = $templates->getData($page);
-
-
-
-if ($page == 'bookingDetails') {
-    if(isset($_POST['postUpdateCancelation'])){
-        $cancelation = $reserSys->cancelBooking($_POST['postChannelId'], $_POST['postBookingId']);
-        $data['content']['bookingDetails'] = $reserSys->forceShowBookingUpdate($_POST['postChannelId'], $_POST['postBookingId']);
+if ($page == 'updateCustomer') {
+    if (isset($_POST['postSearchCustomer'])) {
+        if (isset($_POST['updateCustomer'])) {
+            $reserSys->updateCustomer($_POST['postChannelId'], $_POST['postInfoCustomer']);
+            // error_log(print_r($_POST['postInfoCustomer'], true));
```

Comment:
remove comments which are not about logic

Reply:
fixed